metrics/internal/collectors/storageconsumer.go

```diff
@@ -137,6 +145,10 @@ func (c *StorageConsumerCollector) collectStorageConsumersMetadata(storageConsum
 			prometheus.GaugeValue,
 			float64(encodeVersion(storageConsumer.Status.Client.OperatorVersion)),
 			storageConsumer.Name)
+
+		ch <- prometheus.MustNewConstMetric(c.StorageQuotaUtilizationRatio,
+			prometheus.GaugeValue, storageConsumer.Status.Client.StorageQuotaUtilizationRatio,
+			storageConsumer.Name)
```

Comment:
this is a bad identification for an alert, the consumer name is mainly a hash and we do not display it in the UI. Use the storage client name in conjunction with the cluster name for the user to be able to identify the client.